service/key/key.go

```diff
@@ -267,15 +269,31 @@ func RemoteWritePasswordKey() string {
 // We do not have a provider agnostic label like "giantswarm.io/version" to define this.
```

Comment:
I guess this comment is not needed anymore

Reply:
thanks, fixed